From 22edac84172dfd3a6b06c75ae5e0296cb3a54257 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 6 May 2024 06:17:50 -0500
Subject: [PATCH] Update elasticmatrix.py

---
 .../atomistics/master/elasticmatrix.py        | 29 ++-----------------
 1 file changed, 2 insertions(+), 27 deletions(-)

diff --git a/pyiron_atomistics/atomistics/master/elasticmatrix.py b/pyiron_atomistics/atomistics/master/elasticmatrix.py
index 9c11d03c9..12b622620 100644
--- a/pyiron_atomistics/atomistics/master/elasticmatrix.py
+++ b/pyiron_atomistics/atomistics/master/elasticmatrix.py
@@ -70,38 +70,13 @@ def analyse_structures(self, output_dict):
         Returns:
 
         """
-        (
-            elastic_matrix,
-            self._data["A2"],
-            self._data["strain_energy"],
-            self._data["e0"],
-        ) = analyse_structures_helper(
+        sym_dict, elastic = analyse_structures_helper(
             output_dict=output_dict,
             sym_dict=self._data,
             fit_order=self.fit_order,
             zero_strain_job_name=self.zero_strain_job_name,
         )
-        elastic = ElasticProperties(elastic_matrix=elastic_matrix)
-        self._data.update(
-            {
-                "C": elastic.elastic_matrix(),
-                "S": elastic.elastic_matrix_inverse(),
-                "BV": elastic.bulkmodul_voigt(),
-                "BR": elastic.bulkmodul_reuss(),
-                "BH": elastic.bulkmodul_hill(),
-                "GV": elastic.shearmodul_voigt(),
-                "GR": elastic.shearmodul_reuss(),
-                "GH": elastic.shearmodul_hill(),
-                "EV": elastic.youngsmodul_voigt(),
-                "ER": elastic.youngsmodul_reuss(),
-                "EH": elastic.youngsmodul_hill(),
-                "nuV": elastic.poissonsratio_voigt(),
-                "nuR": elastic.poissonsratio_reuss(),
-                "nuH": elastic.poissonsratio_hill(),
-                "AVR": elastic.AVR(),
-                "C_eigval": elastic.elastic_matrix_eigval(),
-            }
-        )
+        self._data.update(elastic)
 
     @staticmethod
     def subjob_name(i, eps):
