From ab84eb0a9215622a73921a30a025bd8a9b8875f0 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 10 Nov 2025 12:40:34 +0100
Subject: [PATCH] LAMMPS: Add support for compute_modify

---
 pyiron_atomistics/lammps/control.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyiron_atomistics/lammps/control.py b/pyiron_atomistics/lammps/control.py
index c1484ff76..aff036369 100644
--- a/pyiron_atomistics/lammps/control.py
+++ b/pyiron_atomistics/lammps/control.py
@@ -62,6 +62,7 @@ def _init_block_dict(self):
         )
         block_dict["compute"] = (
             "compute",
+            "compute_modify",
             "fix",
             "variable",
             "fix_modify",
