From 99db4a1c498f8f95224247204c9ec642c77e4913 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 15 Sep 2025 11:09:18 +0200
Subject: [PATCH] fix mpi4py compatibility

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0b99d37..740b148 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -39,7 +39,7 @@ dependencies = [
     "pyiron_lammps>=0.3.2,<=0.4.6",
     "pyiron_snippets>=0.1.3,<=0.2.0",
     "pyiron_vasp>=0.2.2,<=0.2.12",
-    "pylammpsmpi>=0.2.18,<=0.3.0",
+    "pylammpsmpi>=0.2.18,<=0.3.1",
     "pyscal3>=3.2.5,<=3.2.7",
     "scipy>=1.15.0,<=1.15.2",
     "scikit-learn>=1.2.1,<=1.6.1",
