From 8132dad84a109989272d4352c0051bfa099520f7 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 9 Oct 2025 16:41:52 +0200
Subject: [PATCH] Fix vasp function

---
 pyiron_atomistics/vasp/vasp.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyiron_atomistics/vasp/vasp.py b/pyiron_atomistics/vasp/vasp.py
index bcceda1d7..868adadb6 100644
--- a/pyiron_atomistics/vasp/vasp.py
+++ b/pyiron_atomistics/vasp/vasp.py
@@ -226,7 +226,7 @@ def vasp_function(
         job.set_kpoints(**kpoints_kwargs)
     server_dict = job.server.to_dict()
     server_dict.update(server_kwargs)
-    job.server.from_dict(server_dict=server_dict)
+    job.server.from_dict(obj_dict=server_dict)
     if calc_mode == "static":
         job.calc_static()
     elif calc_mode == "md":
