From bf73111042def0523c4ce911babcdcc02509fb64 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 4 Dec 2023 16:40:24 +0100
Subject: [PATCH] Update __init__.py

---
 pyiron_atomistics/__init__.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pyiron_atomistics/__init__.py b/pyiron_atomistics/__init__.py
index 253d772fc..86181cb0b 100644
--- a/pyiron_atomistics/__init__.py
+++ b/pyiron_atomistics/__init__.py
@@ -1,6 +1,3 @@
-__version__ = "0.1"
-__all__ = []
-
 from pyiron_atomistics.project import Project
 from pyiron_atomistics.toolkit import AtomisticsTools
 from pyiron_atomistics.atomistics.structure.atoms import (
